openbas-api/src/main/java/io/openbas/rest/inject/InjectApi.java

```diff
@@ -505,6 +518,19 @@ public Inject scenarioInject(
     return injectRepository.findById(injectId).orElseThrow(ElementNotFoundException::new);
   }
 
+  @Transactional(rollbackFor = Exception.class)
+  @PutMapping(INJECT_URI + "/{scenarioId}/injects/{injectId}/bulk")
+  @PreAuthorize("isScenarioPlanner(#scenarioId)")
+  public Inject bulkUpdateInjectForScenario(
+      @PathVariable String scenarioId,
+      @PathVariable String injectId,
```

Comment:
I'm gonna be super annoying but in a bulk update endpoint, shouldn't we send the list of injects to update in one call instead of having many calls viewed as bulk ? Since this endpoint is dedicated to those kind of calls, that would make sense and prevent us from calling a lot of times the API for almost the same call with just the inject id changing ...

Reply:
You are right! I'll modify it, thank you!